tags_exec.go

```diff
@@ -0,0 +1,45 @@
+package pongo2
+
+import (
+	"bytes"
+)
+
+type tagExecNode struct {
+	position    *Token
+	bodyWrapper *NodeWrapper
+}
+
+func (node *tagExecNode) Execute(ctx *ExecutionContext, writer TemplateWriter) *Error {
+	temp := bytes.NewBuffer(make([]byte, 0, 1024)) // 1 KiB size
+
+	err := node.bodyWrapper.Execute(ctx, temp)
+	if err != nil {
+		return err
+	}
+	templateSet := ctx.template.set
+	s := temp.String()
+	currentTemplate, err2 := templateSet.FromString(s)
```

Comment:
Not need to have differrent name for err. you can use as err instead of err2

Reply:
issue is that err and err2 are of different types.. err is basically pongo2.Error. thats why it is causing type error